.github/workflows/kickstart-tests.yml

```diff
@@ -180,6 +183,87 @@ jobs:
           echo "::endgroup::"
           EOF
 
+      - name: Build RHEL-9 boot.iso
+        if: needs.pr-info.outputs.base_ref == 'rhel-9'
+        run: |
+          # /var/tmp tmpfs speeds up lorax and avoids https://bugzilla.redhat.com/show_bug.cgi?id=1906364
+          sudo podman run -i --rm --privileged --tmpfs /var/tmp:rw,mode=1777 -v `pwd`:/source:ro -v `pwd`/kickstart-tests/data/images:/images:z ${{ env.LORAX_BUILD_CONTAINER }} <<EOF
+          set -eux
+
+          # lorax must run on RHEL host to produce RHEL guest images :-(
+          # need to enable internal repos so that we don't need a subscription;
+          rm -f /etc/yum.repos.d/*
+          cat <<EOR > /etc/yum.repos.d/rhel9.repo
+          [RHEL-9-BaseOS]
+          name=baseos
+          baseurl=http://download.devel.redhat.com/rhel-9/nightly/RHEL-9-Beta/latest-RHEL-9/compose/BaseOS/\\\$basearch/os/
```

Comment:
I don't doubt it works as it should, but I don't understand it. Why are there three `\`s ?

Reply:
Yep, it's a bash nonsense.